packages/components/package.json

```diff
@@ -36,19 +36,18 @@
     "node": ">=14.21.2"
   },
   "dependencies": {
+    "@emotion/cache": "^11.11.0",
+    "@emotion/react": "^11.11.4",
```

Comment:
These dependencies were used in the `StylesInjection.tsx` but not explicitly added as dependencies.